packages/ckeditor5-engine/src/view/datatransfer.ts

```diff
@@ -16,19 +16,34 @@ export default class DataTransfer {
 	/**
 	 * The array of files created from the native `DataTransfer#files` or `DataTransfer#items`.
 	 */
-	public readonly files: Array<File>;
+	private _files: Array<File> | null;
 
 	/**
 	 * The native DataTransfer object.
 	 */
 	private _native: DomDataTransfer;
 
-	constructor( nativeDataTransfer: DomDataTransfer ) {
-		this.files = getFiles( nativeDataTransfer );
+	constructor( nativeDataTransfer: DomDataTransfer, options: { cacheFiles?: boolean } = {} ) {
```

Comment:
Doesn't the new option need to be documented?

Or it's blocked by the issue about porting API docs to TSDoc?

Reply:
It's not blocked.

But, IMO, if the parameter name makes its purpose obvious, do we really need docs every time?

Reply:
Second thought. It's not obvious, the cashing happens anyway. It should be called `eagerlyGetFiles` or `lazyGetFiles`, `cacheItNow` or something in this direction.

Reply:
I'd say that while the name says what it does. It does not say anything about why you may need that. And that's what the documentation should cover in most cases too.